admin/classes/rtbiz-hd-email-notification/class-rtbiz-hd-email-notification.php

```diff
@@ -250,7 +250,10 @@ public function notification_new_followup_added( $comment, $comment_type, $uploa
 			}
 
 			$bccemails = $this->exclude_author( $bccemails, $comment->comment_author_email );
-			$bccemails  = apply_filters( 'rtbiz_hd_filter_adult_emails', $bccemails, $comment->comment_post_ID );
+
+			// Get unique emails.
+			$bccemails = array_map( 'unserialize', array_unique( array_map( 'serialize', $bccemails ) ) );
```

Comment:
can you check if can use WP function `maybe_unserialize` and `maybe_serialize`

Reply:
@milindmore22 

I think i've tried that change also and there was some issue. That's why kept as above, which is working perfectly fine.